components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -155,6 +155,29 @@ spec:
 
           - name: TYK_DB_REDISADDRS
             value: {{ include "tyk-dashboard.redis_url" . | quote }}
+
+          - name: TYK_DB_REDISSSLINSECURESKIPVERIFY
+            value: "{{ .Values.global.redis.sslInsecureSkipVerify }}" 
+        
+          {{- if .Values.global.redis.useSSL }}
+          - name: TYK_DB_REDISSSLMAXVERSION
+            value: "{{ .Values.global.redis.sslMaxVersion }}"
+          - name: TYK_DB_REDISSSLMINVERSION
```

Comment:
i am not sure but i do not see any environment variable reference regarding TYK_DB_REDISSSLMINVERSION in tyk-docs. For double check, is the environment variable correct?

Reply:
Maybe it hasn't been updated. [Here](https://github.com/TykTechnologies/storage/blob/24ad76c314348efafedec2c77dd7ab5f51874eb0/temporal/model/config.go#L52) is the value in the TLS config